CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ PowerModels.jl Change Log
 - Added support for unit conversion in transformer entries of PTI files
 - Added support for transformer magnetizing admittance from PTI files
 - Added support for extending PowerModels data with all PTI data fields
+- Added support for Voltage Source Converters and Two-Terminal DC lines in PTI files
+- Added "pti_id" to uniquely identify each component imported from a PTI file
```

Comment:
On second thought, what do you think about calling this `source_id`?  I am thinking it would be more consistent with `source_type` and `source_version`, and we can make it a standard field that other translators use.

Reply:
I agree, `source_id` is a better name for this field to make it generalized to any filetype